.github/workflows/test-benchmark.yml

```diff
@@ -122,7 +124,8 @@ jobs:
         with:
           name: pr
           path: ./pr-id.txt
+          overwrite: true
 
       - name: "Set job summary"
         run: |
-          echo "${{ env.COMMENT_CONTENT }}" >> $GITHUB_STEP_SUMMARY
+          echo "$(<./var/phpbench/summary.txt)" >> $GITHUB_STEP_SUMMARY
```

Comment:
This adds details back to the summary on the action summary page: https://github.com/flow-php/flow/actions/runs/10133264368?pr=1150